python_modules/dagster/setup.py

```diff
@@ -99,7 +99,7 @@ def get_version() -> str:
             "grpcio-tools>=1.32.0,<1.44.0",  # related to above grpcio pins
             "mock==3.0.5",
             "objgraph",
-            "protobuf==3.13.0",  # without this, pip will install the most up-to-date protobuf
+            "protobuf==3.20.3",  # without this, pip will install the most up-to-date protobuf
```

Comment:
Note: this is updating the dagster module itself.

Reply:
Without this change, tests won't run due to incompatible versions. I can move this out to another PR. Should I do this? 
Then we merge that single line change. Before this one. 
@yuhan

Reply:
ah i see. lets include it then.

Reply:
ok finally removed the pin here in a separate PR: https://github.com/dagster-io/dagster/pull/11974